content/millennium/r4/request-and-response/communication.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Communication | R4 API
+---
+
+# Communication
+
+* TOC
+{:toc}
+
+## Overview
+
+The Communication resource is a conveyance of information from one entity, a sender, to another entity, a receiver. The information includes encoded data and optionally a related Patient and/or a related Encounter.
```

Comment:
This shouldn't be and/or because the if the Encounter is provided, the subject (Patient) also must be. Probably just 'and' would be fine.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/433/commits/09eadbbe6726985d8b4f763ec3589c828180e921#diff-a46637e8cbba5ebbd797adadefc1bc2eL12